### ISSUE TITLE
My device SEE

### ISSUE THREAD

**Issue by niquini** [Samsung/SamsungAutomationStudio#50]
will you release the my device see node?

**Comment by iamjaehun** [comment]
Yes, we are working on it to be included in the next release version.

**Comment by iamjaehun** [comment]
The _My device SSE_ node will only be available in [Samsung Automation Studio](https://sasm.developer.samsung.com) for a few months.
Whether and when it will be released is under review, so we can't say for sure yet.
We will let you know as soon as the review is complete.

**Comment by mepagg** [comment]
is there any prediction of when the My device See node will be released in the Samsung Automation pro ndered palette?

**Comment by GED8006** [comment]
I am also waiting for this new feature. Is a release date planned in node red palette?
